test/e2e/specs/specs-calypso/wp-invite-users__editor-spec.js

```diff
@@ -100,6 +101,10 @@ describe( `[${ host }] Invites - New user as Editor: (${ screenSize }) @parallel
 	it( 'As the original user can see and remove new user', async function () {
 		await new LoginFlow( this.driver ).loginAndSelectPeople();
 
+		// Makes sure that the Quick Switch modal will be dismissed.
+		const guideComponent = new GuideComponent( this.driver );
```

Comment:
Component object models are initialized asynchronously so they should be instantiated via the `Expect` call:

```js
const guideComponent = await GuideComponent.Expect( this.driver );
```